### PR TITLE
chore: document available manifests in readme

### PR DESCRIPTION
**Pull request by kukushking**
## Checklist before requesting a review

- [x] I updated `CHANGELOG.MD` with a description of my changes
- [x] If the change was to a module, I ran the code validation script (`scripts/validate.sh`)
- [x] If the change was to a module, I have added thorough tests
- [x] If the change was to a module, I have added/updated the module's README.md
- [x] If a module was added, I added a reference to the module to the repository's README.md
- [x] I verified that my code deploys successfully using `seedfarmer apply`

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by malachi-constant** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: AIOpsModuleTestsInfrastruct-doMLXEYsnmxr
* Commit ID: de4f258dca66c208f32cf2d70754fbe61eeb1f33
* Result: SUCCEEDED
* [Build Logs](https://ythfngxmh6.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=e15c0700-659c-4324-8e3f-78313a233d8d%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

File: README.md [hunk 1]
```diff
@@ -19,6 +19,18 @@ All modules in this repository adhere to the module structure defined in the the
 
 See deployment steps in the [Deployment Guide](DEPLOYMENT.md).
 
+## Project Manifests
+
+End-to-end example use-cases built using modules in this repository.
+
+| Type                                                                                                  | Description                                                                                                                                                                                                  |
+|-------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [MLOps with Amazon SageMaker](manifests/sagemaker/mlops-sagemaker/)                                   | Set us environment for MLOps with Amazon SageMaker. Deploy secure Amazon SageMaker Studio Domain, and provisions SageMaker Project Templates using Service Catalog, including model training and deployment. |
```
Comment (EthanBunce) [inline README.md:28]: Nit: Should this read "Set up environment" instead of "Set us environment"?

Comment (kukushking) [reply README.md]: Corrected, thanks!

**Comment by malachi-constant** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: AIOpsModuleTestsInfrastruct-doMLXEYsnmxr
* Commit ID: 7dba44a56e7c348abdca49b65227c83bc028f86a
* Result: SUCCEEDED
* [Build Logs](https://ythfngxmh6.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=6df9dd4f-85d5-42a0-a89a-14a366c88882%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Review by EthanBunce — APPROVED** [review]

**Pull request merged**